scripts/azure-dev/AzureDev.Ssh.psm1

```
@@ -0,0 +1,435 @@
+
+    [string]$OverrideCidr
+  )
+
+  $cidr = if (-not [string]::IsNullOrWhiteSpace($OverrideCidr)) {
+    $OverrideCidr
+  } else {
+    $Config.AllowedSshCidr
+  }
+
+  if ($Config.ConnectivityMode -eq 'tailscale') {
+    return '10.0.0.0/32'
```

PSScriptAnalyzer (1.25.0)

[info] The cmdlet 'Get-AzureDevAllowedSshCidr' returns an object of type 'System.String' but this type is not declared in the OutputType attribute.

(PSUseOutputTypeCorrectly)